### ISSUE TITLE
nr.db was found．

### ISSUE THREAD

**Issue by SanshiroTakahashi** [jtamames/SqueezeMeta#513]
Sorry for the interruption from Japan.

I have found that
test_install.pl
is now working with the
nr.db
is not able to find.

I tried the same thing with SqueezeMeta-1.4.0 does the same thing, 
BUT the error occurs in SqueezeMeta-1.5.2 is nr.db not found.

This error is caused by a bug in SqueezeMeta-1.5.2
(I haven't had time to track down the details).

One way to fix it is to contact
https://github.com/jtamames/SqueezeMeta
One way to fix it is to look at the end of
15.License and third-party software section.
AND Let's install below software until SqueezeMeta-1.5.2's test_install.pl

trimmomatic
Megahit
Spades
canu
prinseq
kmer-db
cd-hit
amos
mummer
hmmer
barrnap
aragorn
prodigal
DIAMOND
bwa
minimap2
bowtie2
MaxBin
MetaBAT
CONCOCT
DAS tool
checkm
comparem
MinPath
RDP classifier
pullseq
Short-Pair
SAMtools
Mothur
Flye

If you install these first
test_install.pl
can find nr.db and it works fine.

Many similar problems have been posted, but all can be solved in the same way.

Sorry for the sidebar from Japan.

**Comment by fpusan** [comment]
Hi! I understand that you managed to get it working so I will close the issue.
I would like to remark that you shouldn't need to install anything else if you use conda/mamba for installing SqueezeMeta
Just run
```
mamba create -n SqueezeMeta -c conda-forge -c bioconda -c fpusan squeezemeta
conda activate SqueezeMeta
download_databases.pl /path/to/store/databases
test_install.pl
```
This should be enough

**Comment by SanshiroTakahashi** [comment]
> I would like to add that if you use **conda/mamba** to install SqueezeMeta, you do not need to install anything else.

If I may add that this is the focus of your respondents.
If you use **conda/mamba** to install SqueezeMeta, and
**nr.db**
cannot be built, the answer is that the normal files are missing or destroyed, that it cannot be reproduced in your environment, or that the download is incomplete.

This is not the answer sought by the questioner.

The questioner **states** that
**nr.db**
and are people who want to build databases and use SqueezeMeta-1.5.2 but are about to give up, and
The person asking the question is
nr.db
and are looking for a way to successfully build and deploy nr.db.

In other words.
**conda/mamba** will do it.
It makes no sense to respond to me with something like
if you use conda/mamba, if you start by compiling the source code
and should **clarify** **what is different and how that is different**.

For example, if you, the respondents, have installed enough necessary software, the problem will not be a problem.
The questioner is not.
Do you understand this difference and are you speaking?
